.build/parent-pom-template.xml

```diff
@@ -774,6 +774,13 @@
       </dependency>
       <!-- end of chronicle-queue -->
 
+      <dependency>
+        <groupId>software.amazon.cryptools</groupId>
+        <artifactId>AmazonCorrettoCryptoProvider</artifactId>
+        <version>2.2.0</version>
+        <classifier>linux-x86_64</classifier>
```

Comment:
@ayushisingh29 what other classifiers there are? I think there is also classifier for arm? So what does this mean for the project? Because if we depend here on x86, what about people who want to run Cassandra on arm?

Reply:
The other classifier is `linux-aarch_64` which can be used to use ACCP on Linux ARM64 platforms. It is necessary to provide the classifier otherwise we will get an empty package. In this case it will not work for ARM.

Reply:
yeah, check the discussion on the ticket, I am afraid we can not ship anything architecture specific. We need to keep vanilla Cassandra architecture agnostic.